git/git.go

```diff
@@ -307,6 +307,19 @@ func CheckoutBranch(branch string) error {
 	return run.PrepareCmd(configCmd).Run()
 }
 
+// pull changes from remote branch without version history
+func RunPull(baseRemote, baseBranch string) error {
```

Comment:
Nit: there is no need to prefix a function with "Run". Calling any function is, in essence, "running" it. It can just be called `git.Pull()`